.github/workflows/ci.yml

```diff
@@ -2,16 +2,17 @@ name: Ruby
 
 on:
   push:
-    branches: [ master ]
   pull_request:
     branches: [ master ]
+  schedule:
+    - cron: '0 0 1 * *'
 
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.7.2', '3.0']
+        ruby-version: ['2.7.2', '3.0', '3.1', 'head']
```

Comment:
As the gemspec already has upper boundary of `< 3.2` I don't think we should include `head` here